CMakeLists.txt

```diff
@@ -100,10 +100,10 @@ if ( T8CODE_BUILD_BENCHMARKS )
     add_subdirectory( ${CMAKE_CURRENT_LIST_DIR}/benchmarks )
 endif()
 
-if ( BUILD_DOCUMENTATION ) 
+if ( T8CODE_BUILD_DOCUMENTATION )
     add_subdirectory( ${CMAKE_CURRENT_LIST_DIR}/doc )
 endif()
 
-if ( BUILD_SPHINX_DOCUMENTATION )
-    set (BUILD_DOCUMENTATION 1)
+if ( T8CODE_BUILD_DOCUMENTATION_SPHINX )
+    set( T8CODE_BUILD_DOCUMENTATION 1 )
```

Comment:
Please consider using this: https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html

Reply:
No worries, I think it makes total sense! I'll do that right away.

Reply:
It's done. Now the option `T8CODE_BUILD_DOCUMENTATION_SPHINX` is hidden to the user in ccmake until `T8CODE_BUILD_DOCUMENTATION` is set to `ON`. Setting sphinx documentation on the command line now requires both `T8CODE_BUILD_DOCUMENTATION` and `T8CODE_BUILD_DOCUMENTATION_SPHINX` to be turned on.